src/structured-data/structured-data-tags.js

```diff
@@ -578,6 +578,10 @@ export function StructuredDataTags(
   }
 
   if (!isStructuredDataEmpty && pageType === 'story-page') {
+    if (structuredData.enableVideo && story['story-template'] === 'video') {
```

Comment:
You can remove these conditions. Please add it if the organisation object is sent.